chapters/en/_toctree.yml

```diff
@@ -167,6 +167,27 @@
     title: End-of-chapter quiz
     quiz: 8
 
+- local: chapter9
+  title: 9. Building and sharing demos
+  new: true
```

Comment:
FYI this field will eventually render a "New" icon on the frontend - see internal Slack thread: https://huggingface.slack.com/archives/C01SJL5C5LK/p1651736595622049

Reply:
@mishig25 should I expect to see the "new" icon already in this PR or is this something that will become visible once I re-build the content on the `release` branch (for prod)?

<img width="292" alt="Screen Shot 2022-05-10 at 11 53 40" src="https://user-images.githubusercontent.com/26859204/167601878-c2e56f20-cba5-4e47-b66b-4e166228bc32.png">

Reply:
Sounds good